packages/frontend/src/components/ProjectAccess/CreateProjectAccessModal.tsx

```diff
@@ -28,68 +21,59 @@ import MantineIcon from '../common/MantineIcon';
 
 interface Props {
     projectUuid: string;
+    roles: { value: string; label: string; group: string }[];
     onClose: () => void;
 }
 
-const CreateProjectAccessModal: FC<Props> = ({ projectUuid, onClose }) => {
+const CreateProjectAccessModal: FC<Props> = ({
+    projectUuid,
+    roles,
```

Comment:
we are passing now the full list of roles from the parent object into this  modal